src/state/Maps/actions.js

```diff
@@ -24,6 +26,77 @@ import SelectionsAction from '../Selections/actions';
  * CREATORS
  * ================================================== */
 
+/**
+ * Add map to store
+ * @param mapState {Object}
+ */
+const addMap = mapState => {
+	return (dispatch, getState) => {
+		if (!mapState) {
+			dispatch(commonActions.actionGeneralError(`No map state given`));
+		} else if (!mapState.key) {
+			dispatch(
+				commonActions.actionGeneralError(`Undefined mapKey for map ${mapState}`)
+			);
+		} else {
+			dispatch(actionAddMap(mapState));
+			dispatch(use(mapState.key, null, null));
```

Comment:
Add to documentation when "use" is calling.

I mean to functions addMap, addMapToSet, addMapSet

Reply:
And remove...

Reply:
> Add to documentation when "use" is calling.
> 
> I mean to functions addMap, addMapToSet, addMapSet

Where?

Reply:
> And remove...

What?

Reply:
https://github.com/gisat-panther/ptr-state/pull/87/commits/2b603df1cdd9bd5c250b7f746fa36d57075979ba